pyproject.toml

```diff
@@ -33,9 +33,6 @@ dependencies = [
   "uvloop~=0.19.0; sys_platform != 'win32' and platform_python_implementation == 'CPython' and platform_machine != 'armv7l'"
   ]
 
-[project.scripts]
```

Comment:
Hey @patrick91 👋 

Thank you for the PR 😄 . But turns out `[project.scripts]` key is the one required for `maturin` based projects. 

`[tool.poetry.scripts]` will only work for poetry-based publishing system.

Reply:
@sansyrox so then I'm not sure why it doesn't work, how do you build and publish the project?

Reply:
@patrick91 , oh, it works now 😅 See this - https://github.com/sparckles/Robyn/releases/tag/v0.49.3

> how do you build and publish the project?

we use `maturin build` command with different flags based on the architecture and python version.

Reply:
what about publishing?

Reply:
@patrick91 , we use `twine` https://github.com/sparckles/Robyn/blob/04291e9f558c2d2d995f98bab1a9e1440331150b/.github/workflows/release-CI.yml#L177